src/routes/routes.js

```diff
@@ -4,12 +4,10 @@ const router = express.Router();
 const newUser = require('../controllers/userController');
 const upload = require('../controllers/upload');
 
-// GET response for '/'
+// GET response for '/' and redirect to documentation page
 router.get('/', (req, res) => {
     
-    res.status(200).json({ 
-        msg: "Welcome to Dockerized User Management App [by TEAM GRANITE]" 
-    });
+    res.status(200).redirect('/api-docs');
```

Comment:
Why do we have to redirect it?

Reply:
Not to clutter the entry point with much code or use a separate html doc for that purpose.

Reply:
The instruction from Mark is to have the base URL show the swagger doc, we can't redirect it.